api/api.go

```diff
@@ -398,8 +398,16 @@ type CloudBackupGenericRequest struct {
 	// MetadataFilter indicates backups whose metadata has these kv pairs
 	MetadataFilter map[string]string
 	// CloudBackupID must be specified if one needs to enumerate known single
-	// backup( format is clusteruuidORBucketName/srcVolId-SnapId(-incr)
+	// backup( format is clusteruuidORBucketName/srcVolId-SnapId(-incr). If t\
```

Comment:
nit: space after backup and no space after '('.

// backup (format is clusteruuidORBucketName/SrcVolumeID-SnapId(-incr).
// If specified everything else in the request is ignored

Reply:
Done